### PR TITLE
TW 648 post-v10 Security tab screenshot cleanup

### PR DESCRIPTION
**Pull request by HeatherStenson**
I did a sweep of images that show a request response and that are missing the Security tab. 🧹

### CODE REVIEW

**Review by petewitcher — APPROVED** [review]
LGTM!

File: src/pages/docs/designing-and-developing-your-api/mocking-data/mock-with-api.md [hunk 1]
```diff
@@ -70,7 +70,7 @@ If you send the request, you'll receive an authentication error. To [authenticat
 
 Select **Send** to send the `GET All Collections` request. The response pane displays a list of all your collections. Search for the `testAPI` collection and locate the `uid` value. You will use this collection ID in the next step.
 
-<img alt="Getting the collection ID" src="https://assets.postman.com/postman-docs/mock-api-collection-id-v9-14.jpg" width="893px">
+<img alt="Getting the collection ID" src="https://assets.postman.com/postman-docs/v10/mock-api-get-collection-id-v10.jpg" width="750px">
```
Comment (MatthewPinsker-postman) [inline src/pages/docs/designing-and-developing-your-api/mocking-data/mock-with-api.md:73]: Should we blur the entire numeric API key value (`PMAK-***`) for security?

Comment (HeatherStenson) [reply src/pages/docs/designing-and-developing-your-api/mocking-data/mock-with-api.md:73]: I deleted the API key - if someone goes to the trouble of trying to use it, they'll be very disappointed! 😁

File: src/pages/docs/designing-and-developing-your-api/mocking-data/mock-with-api.md [hunk 2]
```diff
@@ -106,15 +106,15 @@ Next, add the following **raw** JSON code to the **Body** tab of the request, su
...
+<img alt="Creating a mock server" src="https://assets.postman.com/postman-docs/v10/mock-api-create-mock-v10.jpg" width="750px">
 
 ## Step 4: Get the mock server URL
 
 To send a request to your mock server, you need to know the mock server URL. You can retrieve the mock server URL using the [GET All Mocks](https://documenter.getpostman.com/view/12959542/UV5XjJV8?_ga=2.100201950.1771040895.1644854022-1154140310.1627600155#52a729a4-4a36-46e5-bfdf-fda0af228d2f) endpoint. Create a new request in Postman, leave `GET` selected for the method, and enter the following URL: `https://api.getpostman.com/mocks`
 
 Add an `x-api-key` header with your Postman API Key, and then select **Send**. The response pane displays a list of all your mock servers. Search for the `testAPImock` mock server and locate the `mockUrl` value. You'll use this URL to send a request to the mock server.
 
-<img alt="Getting the mock server URL" src="https://assets.postman.com/postman-docs/mock-api-mock-url-v9-14.jpg" width="893px">
+<img alt="Getting the mock server URL" src="https://assets.postman.com/postman-docs/v10/mock-api-mock-url-v10.jpg" width="750px">
```
Comment (MatthewPinsker-postman) [inline src/pages/docs/designing-and-developing-your-api/mocking-data/mock-with-api.md:117]: Same question as line 73.

File: src/pages/docs/designing-and-developing-your-api/mocking-data/mock-with-api.md [hunk 3]
```diff
@@ -80,7 +80,7 @@ Create a new request in Postman, leave `GET` selected for the method, and enter
 
 Make sure to add an `x-api-key` header with your Postman API Key, and then select **Send**. The response pane displays a list of all your environments. Search for the `testAPIenv` environment and locate the `uid` value.
 
-<img alt="Getting the environment ID" src="https://assets.postman.com/postman-docs/mock-api-environment-id-v9-14.jpg" width="893px">
+<img alt="Getting the environment ID" src="https://assets.postman.com/postman-docs/v10/mock-api-environment-id-v10.jpg" width="750px">
```
Comment (MatthewPinsker-postman) [inline src/pages/docs/designing-and-developing-your-api/mocking-data/mock-with-api.md:83]: Same question as line 73.

**Review by MatthewPinsker-postman — APPROVED** [review]
Looks good to me!

**Pull request merged**